docs/public/specifications/tests/Address-TP/address01.md

```diff
@@ -10,38 +21,120 @@ name servers must have addresses in the routable public addressing space.
 
 IANA is responsible for global coordination of the IP addressing system.
 Aside its address allocation activities, it maintains reserved address ranges
-for special uses. These ranges can be categorized into three types : 
-[Special purpose IPv4
-addresses](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xml),
-[Special purpose IPv6
-addresses](https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xml)
-and [Multicast reserved
-addresses](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xml).
+for special uses. These ranges can be categorized into two types: 
+[Special purpose IPv4 addresses] and [Special purpose IPv6 addresses].
+
+### Scope
+
+IP addresses from authoritative name servers are matched against IANAs list of 
+public routable address ranges.
 
 ### Inputs
 
-The domain name to be tested.
+* The domain name to be tested.
+
+### Summary
+
+Message Tag                       | Level    | Arguments | Message ID for message tag
+:-------------------------------- |:---------|:----------|:--------------------------
+A01_ADDR_GLOBALLY_REACHABLE       | INFO     |           | All IP addresses of all name servers are in the globally reachable address space.
+A01_NO_GLOBALLY_REACHABLE_ADDR    | CRITICAL |           | None of the name servers IP addresses listed as globally reachable.
+A01_ADDR_NOT_GLOBALLY_REACHABLE   | ERROR    | ns_list   | IP address for "{ns_list}" not listed as globally reachable.
+A01_DOCUMENTATION_ADDR            | ERROR    | ns_list   | IP address for "{ns_list}" part of address range for documentation purposes.
+A01_LOCAL_USE_ADDR                | ERROR    | ns_list   | IP address for "{ns_list}" part of address range intended for local use on network or service provider level..
 
-### Ordered description of steps to be taken to execute the test case
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [Argument list].
+
+### Test procedure 
 
 1. Obtain the IP addresses of each name server of the domain from the parent using
-   [Method4](../Methods.md) and child using [Method5](../Methods.md)
+   [Method4] and [Method5]. 
```

Comment:
We should now switch to methods in [MethodsV2] whenever we do an update of an specification.

[MethodsV2]: https://github.com/zonemaster/zonemaster/blob/master/docs/public/specifications/tests/MethodsV2.md